packages/theme/brand/altinn.css

```
@@ -1273,7 +1273,7 @@
  */
 
 @layer ds.theme.color {
-:root, [data-color="accent"] {
+:root, .ds-avatar, .ds-button, .ds-link, [data-color="accent"] {
```

can we avoid this and rather do something in the respective component CSS files?
```suggestion
:root, [data-color="accent"] {
```